docs/scaling.md

```diff
@@ -12,12 +12,37 @@ As of v3.0, the controller supports having a hot-standby for [High Availability]
 
 ## Vertically Scaling
 
-You can scale the controller vertically:
+You can scale the controller vertically in these ways:
 
-- If you have many workflows, increase `--workflow-workers` and `--workflow-ttl-workers`.
-- Increase both `--qps` and `--burst`.
+### Container Resource Requests
 
-You will need to increase the controller's memory and CPU.
+If you observe the Controller using its total request CPU or memory, you should increase those.
+
+### Adding Goroutines to Increase Concurrency
+
+If you have sufficient CPU you can take advantage of it with more goroutines:
+
+- If you have many Workflows and you notice they're not being reconciled fast enough, increase `--workflow-workers`.
+- If you're using `TTLStrategy` in your Workflows and you notice they're not being deleted fast enough, increase `--workflow-ttl-workers`.
+- If you're using `PodGC` in your Workflows and you notice the Pods aren't being deleted fast enough, increase `--pod-cleanup-workers`.
+
+>= v3.5
```

Comment:
```suggestion
> v3.5 and after
```

Blockquote is the stylistic convention used in most of the docs ([random example](https://github.com/argoproj/argo-workflows/blob/8e01696a9c63f25b3660ed479c6f7a57bb61d82d/docs/rest-api.md?plain=1#L5))